model_compression_toolkit/metadata.py

```diff
@@ -43,13 +45,20 @@ def create_model_metadata(tpc: TargetPlatformCapabilities,
 def get_versions_dict(tpc) -> Dict:
     """
 
-    Returns: A dictionary with TPC and MCT versions.
+    Returns: A dictionary with TPC, MCT and TPC-Schema versions.
 
     """
     # imported inside to avoid circular import error
     from model_compression_toolkit import __version__ as mct_version
-    tpc_version = f'{tpc.name}.{tpc.version}'
-    return {MCT_VERSION: mct_version, TPC_VERSION: tpc_version}
+    tpc_minor_version = f'{tpc.tp_model.tpc_minor_version}'
+    tpc_patch_version = f'{tpc.tp_model.tpc_patch_version}'
+    tpc_platform_type = f'{tpc.tp_model.tpc_platform_type}'
+    tpc_schema = f'{tpc.tp_model.SCHEMA_VERSION}'
+    return {MCT_VERSION: mct_version,
```

Comment:
If you need to access the fields elsewhere, why not define class or named tuple? If this goes directly into the model, no need to define global consts.